skaffold/src/main/kotlin/com/google/container/tools/skaffold/run/SkaffoldRunConfigurationType.kt

```diff
@@ -29,23 +29,13 @@ import javax.swing.Icon
 class SkaffoldRunConfigurationType : ConfigurationType {
     val ID = "google-container-tools-skaffold-run-config"
 
-    override fun getIcon(): Icon {
-        return SKAFFOLD_ICON
-    }
+    override fun getIcon(): Icon = SKAFFOLD_ICON
 
-    override fun getConfigurationTypeDescription(): String {
-        return message("skaffold.run.config.general.description")
-    }
+    override fun getConfigurationTypeDescription(): String = message("skaffold.run.config.general.description")
```

Comment:
is this over the line length limit (I think its 100 for java, I assume its the same for Kotlin)?

note to self: we should add the formatter and static analysis tools soon

Reply:
Done, shortened.

Reply:
[ktlint](https://github.com/shyiko/ktlint) is a great linter for Kotlin (it's used in several of our Android DevRel projects).